infra.bs

```diff
@@ -679,6 +679,20 @@ where <a>UTF-8 encode</a> comes into play.
 
 <hr>
 
+<p>A <a>string</a> <var>a</var> <dfn export>is</dfn> or is <dfn export>identical to</dfn> a 
```

Comment:
We should use `<dfn export for=string lt="is|identical to">is</dfn>` here and then use `<a for=string>is</a>` or `<a for=string>identical to</a>` when referring to it. The `for` attribute helps with disambiguation from other equality comparisons and if we allow multiple terms for the same definition we should list those in the definition so that all the links go to the right place.

Reply:
Addresssed in 774e2c1. Thanks for your help!!